### ISSUE TITLE
Since last update Fan Control window opens when resuming from sleep.

### ISSUE THREAD

**Issue by Ozzuneoj** [Rem0o/FanControl.Releases#1568]
I haven't changed any settings in the application, but after updating to V147 recently the main Fan Control window will be open every time I resume from sleep. If I exit the program completely and start it via the shortcut, it opens minimized to the tray as expected. But if I leave it minimized to tray and then put the computer to sleep, then resume, the window will be open again.

My error log has no entries since December and the program otherwise works fine.

This is on Windows 10 21H2, AMD Ryzen 5800X3D, X570, RTX 3060 Ti.

**Comment by hzmeister** [comment]
I have the same issue.

**Comment by Rem0o** [comment]
Run the updater.exe in fancontrol’s folder again, should fix.

**Comment by Ozzuneoj** [comment]
> Run the updater.exe in fancontrol’s folder again, should fix.

Ah, that did it! I tried it once with Fan Control already running in the tray and it just closed the program and didn't do anything else. After starting it again, closing it manually, then running updater.exe, it opened the fan control window. Then I minimized it, put the PC to sleep, resumed, and it was still minimized to the tray. 

So that was the ticket!

I love this program though. I can't live without it on my Ryzen systems. I'm able to tune my fan curves so that my 5800X3D is basically silent on air (Thermalright FC140) and with all cores set to -30 with PBO2 Tuner I get max boost clocks, better performance and lower temps. Plus, my Dell OEM 3060 Ti has always had a slightly annoying fan pitch at certain speeds and with Fan Control I can basically tune that speed right out so I basically never hear that sound.

All this may be possible with other software, but only Fan Control makes it this easy.
